src/renderer/entities/transaction/ui/Scanning/ScanSingleframeQr.tsx

```diff
@@ -5,7 +5,16 @@ import { UnsignedTransaction } from '@substrate/txwrapper-polkadot';
 import { useI18n } from '@app/providers';
 import { Button, FootnoteText } from '@shared/ui';
 import { WalletIcon } from '@entities/wallet'; // TODO: cross import
-import type { Account, ChainAccount, ChainId, ShardAccount, Wallet, Address } from '@shared/core';
```

Comment:
I'd rather did
`import type { Account, ChainAccount, ChainId, ShardAccount, Wallet, Address } from '@shared/core';`
and
`import { CryptoType } from '@shared/core';`